packages/gatsby/src/utils/graphql-typegen/ts-codegen.ts

```diff
@@ -136,7 +136,6 @@ export async function writeTypeScriptTypes(
       },
       addUnderscoreToArgsType: true,
       skipTypename: true,
-      flattenGeneratedTypes: true,
```

Comment:
Did this due to https://github.com/cometkim/gatsby-plugin-typegen/pull/138#issuecomment-1122652357